bin/test_benchmarks

```diff
@@ -6,6 +6,10 @@ project_dir=$PWD
 rm -f $project_dir/test/bench.output
 touch $project_dir/test/bench.output
 
+rm -f $project_dir/docs/_includes/auto_benchmarks.html
+touch $project_dir/docs/_includes/auto_benchmarks.html
```

Comment:
Just a heads up that the rm/touch pattern is _really_ close to needing to be a loop but it's fine as-is